@3rdweb-sdk/react/hooks/useAnalytics.ts

```diff
@@ -0,0 +1,100 @@
+import { analyticsKeys } from "../cache-keys";
+import { useQueryWithNetwork } from "./query/useQueryWithNetwork";
+import { useContractType } from "@thirdweb-dev/react";
+import { ValidContractClass } from "@thirdweb-dev/sdk";
+import { C } from "ts-toolbelt";
+
+interface Event {
+  eventName: string;
+  dataName: string;
+}
+
+const CONTRACT_ANALYTICS: Record<string, Event[]> = {
+  "nft-collection": [
+    { eventName: "TokensMinted", dataName: "Tokens Minted" },
+    { eventName: "Transfer", dataName: "Tokens Transferred" },
+  ],
+  edition: [
+    { eventName: "TokensMinted", dataName: "Tokens Minted" },
+    { eventName: "TransferSingle", dataName: "Token Transfers" },
+  ],
+  token: [
+    { eventName: "TokensMinted", dataName: "Tokens Minted" },
+    { eventName: "Transfer", dataName: "Tokens Transferred" },
+  ],
+  marketplace: [
+    { eventName: "ListingAdded", dataName: "Listings Created" },
+    { eventName: "NewOffer", dataName: "Offers Made" },
+    { eventName: "NewSale", dataName: "Sales Made" },
+    { eventName: "AuctionClosed", dataName: "Auctions Closed" },
+  ],
+  "nft-drop": [],
+  "edition-drop": [
+    { eventName: "TokensLazyMinted", dataName: "Tokens Minted" },
```

Comment:
This should be "created" instead of "minted" since it's lazyMint. 

do we have a claimed event for edition drop?